modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/odata/search/ODataSearchAdapterImpl.java

```diff
@@ -59,28 +64,16 @@ public Hits search(
 			SearchContext searchContext = _createSearchContext(
 				companyId, start, end);
 
-			Query query = _getQuery(
-				filterString, className, entityModel, filterParser, locale,
-				searchContext);
+			Indexer<Organization> indexer = _indexerRegistry.getIndexer(
```

Comment:
He @darquesdev !!!, there is a typo problem in here with Indexer<Organization> indexer. d you mind changing it and squash it t the original commit?.

Reply:
sure! I'll do it